src/components/markdown.js

```diff
@@ -41,6 +41,35 @@ export const MarkdownViewer = ({ children, renderers, ...props }) => {
   })
 }
 
+export const FirstParagraphMarkdownViewer = ({ children, renderers, ...props }) => {
+  let renderedFirstParagraph = false
+  return h(MarkdownViewer, {
```

Comment:
We should be a little careful here. Several years ago, I _think_ we used to use `marked` for these description previews (on the workspace list page), but we had to stop because `marked` would fail for some content in a way that made the page unusable. It was tricky to track down because it was content-dependent, and I don't know that we ever tracked down exactly what content was causing the problem.

Reply:
We're using `marked` to render workspace descriptions on the workspace dashboard.
https://github.com/DataBiosphere/terra-ui/blob/d20d3f1c48a3d5906e8179dc3a20d9338a93a432/src/pages/workspaces/workspace/Dashboard.js#L386

Reply:
Ah, looks like [AS-643](https://broadworkbench.atlassian.net/browse/AS-643) / #2732 could be the past issue?

Reply:
Added a `toString` in [04fb237](https://github.com/DataBiosphere/terra-ui/pull/3155/commits/04fb237987a478ba9cfc675c3714fc320ad99e76).

Reply:
I believe it was #1980. At one point the workspaces list was a card view rather than a table view and each card would display the description.
IIRC there were certain markdown patterns that, when run through the `marked`, would result in running a regex in one of the dependencies or transitive dep. This would end up in a tight loop crashing the app.